tests/Functional/AirtableClientTest.php

```diff
@@ -8,15 +8,21 @@
 use Yoanbernabeu\AirtableClientBundle\AirtableClient;
 use Yoanbernabeu\AirtableClientBundle\AirtableClientInterface;
 
+/**
+ * @internal
+ */
 class AirtableClientTest extends KernelTestCase
 {
-    public function testIfKernelBootingAndAirtableClientIsRegistered(): void
+    /**
+     * @test
+     */
+    public function ifKernelBootingAndAirtableClientIsRegistered(): void
     {
         $kernel = self::bootKernel();
         $container = $kernel->getContainer();
 
-        $this->assertTrue($container->has(AirtableClientInterface::class));
-        $this->assertTrue($container->has("airtable_client"));
-        $this->assertInstanceOf(AirtableClient::class, $container->get("airtable_client"));
+        static::assertTrue($container->has(AirtableClientInterface::class));
+        static::assertTrue($container->has('airtable_client'));
+        static::assertInstanceOf(AirtableClient::class, $container->get('airtable_client'));
```

Comment:
What the difference between using the context and static ?

Reply:
There is no real difference.
The change comes from the rule [`'php_unit_test_case_static_method_calls' => true`](https://github.com/yoanbernabeu/Airtable-Client-Bundle/pull/8/files#diff-8ec5bae185dc9af5b5bfdcd5f1dcbee62f18525ee488996d52b22729f4076ef0R50).
I can revert that change if you prefer the previous call.

Reply:
Yes there's no difference.
https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.static-vs-non-static-usage-of-assertion-methods

I read that static method in PHPUnit exists to allows developers to use these methods without extending TestCase like an assert library.

Reply:
Ok thanks ;)